frontend/src/components/campAdmin/DialogCategoryCreate.vue

```diff
@@ -56,10 +57,16 @@ export default {
     },
   },
   methods: {
-    createCategory() {
-      return this.create().then(() => {
-        this.api.reload(this.camp.categories())
-      })
+    async createCategory() {
+      const createdCategory = await this.create()
+      await this.api.reload(this.camp.categories())
```

Comment:
what happens if this throws an error?

Reply:
We get a sentry issue. Is this a use case which you would like me to handle specially? The request goes to `/categories?camp=/camps/1a2b3c4d`, which should always be available as long as I'm logged in and as long as the API responds at all: https://github.com/ecamp/ecamp3/blob/devel/api/tests/Api/Categories/ListCategoriesTest.php#L20